verification/curator-service/ui/src/components/App.tsx

```diff
@@ -567,6 +582,11 @@ export default function App(): JSX.Element {
                 >
                     <div className={classes.drawerHeader} />
                     <Switch>
+                        {isJustReader() && (
+                            <Route exact path="/">
+                                <Redirect to="/cases" />
```

Comment:
Could this redirect go inside the existing `<Route exact path="/">` at the bottom of this switch (for non curator/admins)?

Reply:
yes I moved it there now.